lib/features/auth/presentation/bloc/user_config/user_config_state.dart

```diff
@@ -1,9 +1,20 @@
 part of 'user_config_cubit.dart';
-
-abstract class UserConfigState {
+class UserConfigState  extends Equatable{
   final UserConfigModel? userConfigModel;
+  final Map<String, dynamic>? selectedVoice;
+
```

Comment:
You can undo all changes in this file, it won't be required if you use `setUserConfig`

Reply:
copy that